workers/common/src/test/kotlin/common/env/config/EnvironmentConfigLoaderTest.kt

```diff
@@ -383,7 +418,7 @@ class EnvironmentConfigLoaderTest : StringSpec() {
         "Simple environment variable definitions are processed" {
             val helper = TestHelper()
 
-            val config = loadConfig(".ort.env.direct-variables.yml", helper)
+            val config = parseConfig(".ort.env.direct-variables.yml", helper)
```

Comment:
The commit message says "To simplify the tests, this commit breaks it up in two subfunctions", but to be honest, I don't see in this diff how the tests became simpler?

Reply:
If you look in `EnvironmentConfigLoaderTest`, you can see that I wrote some tests that could not be easily written before.

Also the tests now mainly use `parseConfig`, which does not unnecessarily resolve the parsed configuration.
Hence, introducing the functions allows to have tests doing only what needs to be tested.

Reply:
Ok, it sounded to me like also the existing tests would become simpler. Anyway, you did notice that the tests are failing?